src/sidebar/components/tag-editor.js

```diff
@@ -34,6 +36,9 @@ function TagEditor({ onEditTags, tags: tagsService, tagList }) {
     setSuggestionsListOpen(false);
   });
 
+  // Convenient boolean  for ie detection.
```

Comment:
I don't think this comment is needed as the purpose of the code seems pretty obvious.